bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/PowerControlPower.java

```diff
@@ -29,20 +32,44 @@
 @NonNullByDefault
 public class PowerControlPower extends BaseChannelHandler<CommandConfirmation> {
     private final Logger logger = LoggerFactory.getLogger(PowerControlPower.class);
+    private final ConfigProvider configProvider;
+    private ScheduledExecutorService scheduler;
+    private static final int WOL_PACKET_RETRY_COUNT = 10;
+    private static final int WOL_PACKET_RETRY_DELAY_MILLIS = 100;
+
+    public PowerControlPower(ConfigProvider configProvider, ScheduledExecutorService scheduler) {
+        this.configProvider = configProvider;
+        this.scheduler = scheduler;
+    }
 
     @Override
     public void onReceiveCommand(String channelId, LGWebOSHandler handler, Command command) {
-        if (OnOffType.ON == command) {
-            logger.debug(
-                    "Received ON - Turning TV on via API is not supported by LG WebOS TVs. You may succeed using wake on lan binding, please consult lgwebos binding documentation.");
-        }
         if (!handler.getSocket().isConnected()) {
             /*
              * Unable to send anything to a not connected device.
              * onDeviceReady nor onDeviceRemoved will be called and item state would be permanently inconsistent.
              * Therefore setting state to OFF
              */
             handler.postUpdate(channelId, OnOffType.OFF);
+        }
+
+        if (OnOffType.ON == command) {
+            String macAddress = configProvider.getMacAddress();
+            if (macAddress.isEmpty()) {
+                logger.debug(
+                        "Received ON - Turning TV on via API is not supported by LG WebOS TVs. You may succeed using wake on lan (WOL). Please set the macAddress config value in Thing configuration to enable this.");
+            } else {
+                for (int i = 0; i < WOL_PACKET_RETRY_COUNT; i++) {
+                    scheduler.schedule(() -> {
+                        try {
+                            WakeOnLanUtility.sendWOLPacket(macAddress);
+                        } catch (IllegalArgumentException e) {
+                            logger.debug("Failed to send WOL packet: {}", e.getMessage());
+                        }
+                    }, i * WOL_PACKET_RETRY_DELAY_MILLIS, TimeUnit.MILLISECONDS);
+                }
```

Comment:
Thought about storing the scheduled job and cancelling it when the power channel updates to ON. But it takes way more than 1 sec  (10x 100ms) for the TV to become responsive and accept a connection, so that the power channel would be updated. So I think the answer is no.